### PR TITLE
EOS-22209: Removed unused 'HealthMonitorError' import

### PR DESCRIPTION
**Pull request by indrajitzagade**
## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Fix codacy issue for an unused import and message bus instantiation
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes, tested using the workspace.
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Fix codacy issue for an unused import and message bus instantiation
  </code>
</pre>
## Solution
<pre>
  <code>
    Removed unused import and fixed message bus instantiation in the test file
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    test_publisher.py ran successfully with these changes.
  </code>
</pre>

### CODE REVIEW

File: ha/core/health_monitor/health_monitord.py [hunk 1]
```diff
@@ -37,7 +37,6 @@
 from ha.util.message_bus import CONSUMER_STATUS
 from ha.core.config.config_manager import ConfigManager
 from ha.core.health_monitor.monitor_rules_manager import MonitorRulesManager
-from ha.core.health_monitor.error import HealthMonitorError
```
Comment (ArchanaLimaye) [inline ha/core/health_monitor/health_monitord.py:40]: curious to know, how the original PR passed the codacy checks  here ?

File: ha/test/integration/health_monitor/test_publisher.py [hunk 2]
```diff
@@ -43,12 +43,12 @@
         print("Consuming the action event")
         message_consumer = MessageBus.get_consumer(consumer_id="1",
                             consumer_group='test_publisher',
-                            message_types=[message_type])
+                            message_type=message_type)
```
Comment (ArchanaLimaye) [inline ha/test/integration/health_monitor/test_publisher.py:46]: The MessageBus here is from ha.util.message_bus ; and it does not have a get_consumer method
Need to import MessageBus from ha/utils

**Review by ArchanaLimaye — CHANGES_REQUESTED** [review]

File: ha/test/integration/health_monitor/test_publisher.py (hunk 2 above)
Comment (Madhura-08) [reply ha/test/integration/health_monitor/test_publisher.py:46]: I can see, its imported from ha.util and and MessageBus class has a static method get_consumer().

But, earlier the type was getting send as list and now its str. And actual implementation now accepts str. So, was the MessageBus logic got changed  or this was not captured earlier?

File: ha/core/health_monitor/health_monitord.py (hunk 1 above)
Comment (indrajitzagade) [reply ha/core/health_monitor/health_monitord.py:40]: I guess the original author had ignored this error.

File: ha/test/integration/health_monitor/test_publisher.py (hunk 2 above)
Comment (indrajitzagade) [reply ha/test/integration/health_monitor/test_publisher.py]: Updated.

**Review by ArchanaLimaye — APPROVED** [review]

**Review by Madhura-08 — APPROVED** [review]

**Review by ajay-paratmandali — APPROVED** [review]

**Pull request merged**